AudioSinkExample/AudioSinks/AudioSinks-Bridging-Header.h

```diff
@@ -0,0 +1,9 @@
+//
+//  AudioSinks-Bridging-Header.h
+//  AudioSinkExample
+//
+//  Copyright © 2017 Twilio Inc. All rights reserved.
+//
+
+#import "ExampleAudioRecorder.h"
+#import "ExampleSpeechRecognizer.h"
```

Comment:
I'm seeing warnings that `TVIRoom (CallKitIntegration)` is defined multiple times. Am I doing something wrong in the bridging header?